AdyenActions/Components/QRCode/QRCodeActionComponent.swift

```diff
@@ -28,7 +28,9 @@ public final class QRCodeActionComponent: ActionComponent, Cancellable {
     public weak var presentationDelegate: PresentationDelegate?
 
     public weak var delegate: ActionComponentDelegate?
-    
+
+    internal let presenterViewController = UIViewController()
```

Comment:
what's the purpose of this viewController?

Reply:
As I am using ShareableComponent protocol, it asked me to add it to conform to this protocol.

Reply:
hmm, I see, ok 👍